ffi/capi/src/decimal.rs

```diff
@@ -14,13 +14,13 @@ use icu_decimal::FixedDecimalFormat;
 /// Opaque type for use behind a pointer, is [`FixedDecimalFormat`]
 ///
 /// Can be obtained via [`icu4x_fixed_decimal_format_create()`] and destroyed via [`icu4x_fixed_decimal_format_destroy()`]
-pub type ICU4XFixedDecimalFormat<'d> = FixedDecimalFormat<'d, 'static>;
+pub type ICU4XFixedDecimalFormat<'data> = FixedDecimalFormat<'data>;
```

Comment:
I think this should lose the lifetime entirely, actually. I don't think we should expose non-`'static` data providers across FFI just yet.

Reply:
I'd like to check in the PR as-is, and figure this out in a follow-up PR.